modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7732,7 +7733,7 @@ height-maximized=Height Maximized
 height-normal=Height Normal
 help=Help
 help-custom-element=Custom element defines CSS and JavaScript resources that will be rendered into a provided HTML &lt;custom-element&gt;. This provides full integration with the Liferay host page. Each custom element client extension is made available as a widget.
-help-fds-cell-renderer=Frontend data set cell renderer defines a JavaScript function that can render a frontend data set cell content.
+help-fds-cell-renderer=Cell renderer defines a JavaScript function that can render a frontend data set cell content.
```

Comment:
This language key should be `help-cell-renderer`.

Also, if we are removing mentions of FDS, it would probably make sense to remove the remaining one: `Cell renderer defines a JavaScript function that can render cell content.`

Reply:
I think we need some consistency review here. It's true that designs don't mention FDS, nevertheless, this client extension needs FDS as it works enhancing the suite of available cell renderers for that component, and no other. In this sense, "FDS" plays the same role as "Theme" in "Theme CSS", "Theme JS" or "Theme SVG Spritemap" as opposed to other, self-contained extensions ("JS", "CSS", "Custom-element" or "Iframe"). In general, we're being very techy and specific when naming frontend client extensions, imho the FDS name shall remain to provide specificity.

Let me do a last check with @pablo-agulla and Uge

Reply:
FWIW, we will also structure our devexp APIs by "functional domains". So, for example, when a dev is creating a FDS cell renderer it needs to import `@liferay/js-api/data-set`

This is something that we won't be able to change in the future (or if we change it, it will be painful for us and customers), so better get it right from the start.

It may look unrelated to this conversation, but my assumption is that, for each of these functional domains, the objects inside have unambiguous names and objects in different domains with the same name may be disambiguated just by adding the functional name.

According to that, it would be enough to name it "Data Set Cell Renderer" (or FDS, or whatever you name it) to make sure that, if we created another type of Cell Renderer in another different domain, there would be no collision.

I'm sure there are more things than data sets that can have cells :sweat_smile: